package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "ENV=dev nodemon index.js",
     "start": "ENV=prod node index.js",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint \"./*/*\" --fix"
```

Comment:
Can wildcard only .js file here to prevent unnecessary linting to other file types